### PR TITLE
Bakta update to 1.8.1

### PR DESCRIPTION
**Pull request by pimarin**
Update of bakta to 1.8.1 and add the  `--meta`

### CODE REVIEW

**Comment by pimarin** [comment]
@bernt-matthias Is it okay ?

File: tools/bakta/bakta.xml [hunk 1]
```diff
@@ -88,8 +89,8 @@
         <section name="input_option" title="Input/Output options" expanded="true">
             <param name="bakta_db_select" type="select" label="The bakta database">
                 <options from_data_table="bakta_database">
-                  <filter type="static_value" value="@BAKTA_VERSION@" column="bakta_version"/>
-                  <validator message="No bakta database is available" type="no_options"/>
+                    <filter type="static_value" value="@MIN_COMPATIBLE_BAKTA_VERSION@" column="bakta_version"/>
```
Comment (bernt-matthias) [inline tools/bakta/bakta.xml:92]: Just asking: the `MIN` in the token name suggests that you are assuming that the filter allows a comparison .. actually it just allows one exact value.

**Comment by bernt-matthias** [comment]
Looks like we can merge this one and the data manager. Ready?

**Comment by bgruening** [comment]
Ready!

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**